addon/components/listbox/-label.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+
+import { modifier } from 'ember-modifier';
+
+export default class ListboxLabelComponent extends Component {
+  registerLabel = modifier((element) => {
+    this.args.registerLabelElement(element);
```

Comment:
This can be done in a followup, but if `registerLabelElement`, itself, was a modifier, we don't need this class, and we can do `<div {{@registerLabelElement}}>`